src/Bicep.LangServer/LangServerResources.resx

```diff
@@ -120,12 +120,18 @@
   <data name="DeploymentFailedMessage" xml:space="preserve">
     <value>Deployment failed for {0}.</value>
   </data>
+  <data name="DeploymentFailedResult" xml:space="preserve">
```

Comment:
Never localize programmatic values.  The telemetry queries need to always see "Succeeded" and "Failed", not various language versions.

Reply:
I have a PR that adds this to BicepTelemetryEvent.cs:

 public record BicepTelemetryEvent : TelemetryEventParams
    {
        public static class Result
        {
            public const string Succeeded = "Succeeded";
            public const string Canceled = "Canceled";
            public const string Failed = "Failed";
        }

Reply:
Included the Result class in this PR as well:
https://github.com/Azure/bicep/commit/f966ce69fc2547ef8a610340e8cde9a100e5ca0b

Thanks!